src/main/g8/build.sbt

```diff
@@ -1,11 +1,13 @@
-import Dependencies._
-
-lazy val root = (project in file(".")).
-  settings(
-    inThisBuild(List(
-      organization := "com.example",
-      scalaVersion := "2.12.1"
-    )),
-    name := "$name$",
-    libraryDependencies ++= Seq(akkaActor, akkaTestkit % Test, scalaTest % Test)
-  )
+name := "hello-akka-scala"
+
+version := "1.0"
+
+scalaVersion := "2.12.2"
+
+lazy val akkaVersion = "2.5.1"
```

Comment:
We could do that but then we would have to remove the `build.sbt` file from the "running-the-application.md" file as Paradox will just publish the file as it is (and not do any g8 substitution). I am fine with either.